### ISSUE TITLE
[BUG] scripts stops moving after first game

### ISSUE THREAD

**Issue by BeeJayJayn** [waibcam/AutoSave-Highlights-for-Hunt-Showdown#1]
The powershell also doesn't report an error or such, just doesnt show newly found clips anymore. 

Also might be tied to dying and that clip breaking it?

**Comment by waibcam** [comment]
Sorry for late response. I was away and then many things to catch up.

As for your issue, I never had this behavior. Dying shouldn't affect as the script is triggered simply when files are created in the folder you set.

Is this happening all the time or did it happen just once or several times only?

**Comment by BeeJayJayn** [comment]
I can reliable recreate the process by getting a kill AND then die with in a few seconds. Is there a queue?

**Comment by waibcam** [comment]
OK, so far, I see 2 possibilities then:

- line 116: $Timeout = 1000
You can try to increase this value to like "5000" (5 sec). This may improve when 2 files are created closed to each other (less than a second) and the script doesn't have the time to process them and could crash.

- Line 131: Start-Sleep -Seconds 5
Increase this value to "10". This may improve if the file takes too much time to be fully created on your PC.
Basically the script is triggered but when it tries to copy file(s) it hangs because file(s) is(are) still beeing created.

Let me know if that helps!
